README_first.ipynb

```diff
@@ -1,60 +1,160 @@
 {
```

Comment:
<p>"an external that" missing the word model?</p><p> </p><p>also "external" is a bad word for describing a model, external to what? Maybe "and learn how to train a model which is not yet supported by <code>optimum-graphcore</code>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/Graphcore-HuggingFace-fork/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>